draft-ietf-httpbis-cache-latest.xml

```diff
@@ -408,6 +408,10 @@
    before forwarding the message, and this &MAY; be implemented by doing so
    before storage; see <xref target="field.connection"/> for some
    examples.</li>
+   <li>The no-cache (<xref target="cache-response-directive.no-cache"/>) and
+   private (<xref target="cache-response-directive.private"/>) cache
+   directives can have arguments that prevent storage of header fields by all
```

Comment:
```suggestion
   directives can have arguments that prevent storage of fields by all
```

(that's actually a general question; does caching only apply to *header* fields???)

Reply:
It's just header fields in this case; see https://httpwg.org/http-core/draft-ietf-httpbis-cache-latest.html#storing.fields and the directive definitions.